opencog/guile/SchemeSmob.cc

```diff
@@ -45,6 +47,25 @@ void SchemeSmob::init()
 	if (is_inited.test_and_set()) return;
 
 	init_smob_type();
+
+	// If one is importing this class then the functions defined in
+	// .scm files associated with '(opencog)' module should be available.
+	// Assuming installation in a standard path.
+	scm_c_eval_string("(add-to-load-path \"/usr/local/share/opencog/scm\")\n");
```

Comment:
This should be CMAKE_BINARY_INSTALL_DIR or something like that -- wherever cmake installs, which defaults to `/usr/local`